### PR TITLE
Add integration test for additional plugin manifest entries

### PR DESCRIPTION
**Pull request by jtnord**
I erroneously forgot to push the latest changes to #436 at the time which mean the IT for it was still sitting on my local machine only.

Whilst [an ITs](https://github.com/jenkinsci/maven-hpi-plugin/pull/436/files#diff-0f2f097e3b673ee6c7452716ac8a8268a8a65d5c8cdf5631dc9098f9ae2518d2) was adapted there was still no IT to cover a multi-module reactor build - so this IT ensures we have coverage for that.

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/it/metadata-it/verify.groovy [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    
+    assert attributes.getValue('Plugin-Developers').equals('Robert McBobface:bob:bob@example.com,:jane:jane@example.com')
+    assert attributes.getValue('Plugin-License-Name').equals('MIT License')
+    assert attributes.getValue('Plugin-License-Url').equals('https://opensource.org/licenses/MIT')
+    assert attributes.getValue('Plugin-ScmUrl').equals('https://github.com/jenkinsci/maven-hpi-plugin')
+    assert attributes.getValue('Plugin-Version').startsWith('1.0-SNAPSHOT')
+    assert attributes.getValue('Url').equals('https://plugins.jenkins.io/multimodule-it-plugin2/')
+    assert attributes.getValue('Plugin-ScmConnection').equals('scm:git:https://github.com/jenkinsci/maven-hpi-plugin.git')
+    assert attributes.getValue('Plugin-GitHash').length() == 40
+    assert attributes.getValue('Plugin-Module').equals('plugin2') // invoker copies the file so this is where it is actualy built.
```
Comment (basil) [inline src/it/metadata-it/verify.groovy:71]: Ditto

File: src/it/metadata-it/verify.groovy [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    
+    assert attributes.getValue('Plugin-Developers').isEmpty()
+    assert attributes.getValue('Plugin-License-Name').equals('MIT License')
+    assert attributes.getValue('Plugin-License-Url').equals('https://opensource.org/licenses/MIT')
+    assert attributes.getValue('Plugin-ScmUrl').equals('https://github.com/jenkinsci/maven-hpi-plugin')
+    assert attributes.getValue('Plugin-Version').startsWith('1.0-SNAPSHOT')
+    assert attributes.getValue('Url').equals('https://plugins.jenkins.io/plugin1/')
+    assert attributes.getValue('Plugin-ScmConnection').equals('scm:git:https://github.com/jenkinsci/maven-hpi-plugin.git')
+    assert attributes.getValue('Plugin-GitHash').length() == 40
+    assert attributes.getValue('Plugin-Module').equals('plugin1') // invoker copies the file so this is where it is actualy built.
```
[the remark this answers is not in the document]
Comment (jtnord) [reply src/it/metadata-it/verify.groovy:38]: comment is not longer correct - so better to just remove it
```suggestion
    assert attributes.getValue('Plugin-Module').equals('plugin1')
```

File: src/it/metadata-it/verify.groovy (hunk 1 above)
Comment (jtnord) [reply src/it/metadata-it/verify.groovy:71]: comment is no longer correct - so better to just remove it
```suggestion
    assert attributes.getValue('Plugin-Module').equals('plugin2')
```

File: src/it/metadata-it/verify.groovy [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    
+    assert attributes.getValue('Plugin-Developers').isEmpty()
+    assert attributes.getValue('Plugin-License-Name').equals('MIT License')
+    assert attributes.getValue('Plugin-License-Url').equals('https://opensource.org/licenses/MIT')
+    assert attributes.getValue('Plugin-ScmUrl').equals('https://github.com/jenkinsci/maven-hpi-plugin')
+    assert attributes.getValue('Plugin-Version').startsWith('1.0-SNAPSHOT')
+    assert attributes.getValue('Url').equals('https://plugins.jenkins.io/plugin1/')
+    assert attributes.getValue('Plugin-ScmConnection').equals('scm:git:https://github.com/jenkinsci/maven-hpi-plugin.git')
+    assert attributes.getValue('Plugin-GitHash').length() == 40
+    assert attributes.getValue('Plugin-Module').equals('plugin1') // invoker copies the file so this is where it is actually built.
```
Comment (jtnord) [inline src/it/metadata-it/verify.groovy:38]: comment is no longer correct so better to remove it
```suggestion
    assert attributes.getValue('Plugin-Module').equals('plugin1')
```

**Review by basil — APPROVED** [review]

**Pull request merged**